Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.10-alpine
+
+WORKDIR /usr/src/app
+
+RUN pip install --no-cache-dir scrap_engine && \
```

```suggestion
RUN pip install --no-cache-dir scrap_engine playsound pygobject && \
```

I would suggest adding those two dependencies, which have been added with the Sound PR.

---

This wont build, because it needs libs and compilers in the image.
I will see what i can do, hopefully i can do some magic with multistage dockerbuild

---

you can build your container like so and save about 275MB~ over a none multistage image:

```
From alpine:3.15 AS compile-image

RUN apk add \
    python3-dev \
    py3-pip \
    py3-gst \
    gobject-introspection-dev \
    cairo-dev \
    build-base 

RUN python3 -m pip install --user scrap_engine playsound pygobject 

From alpine:3.15

RUN apk add --no-cache \
    python3-dev \
    py3-gst 
    
COPY --from=compile-image /root/.local /root/.local
ENV PATH="/root/.local/bin:$PATH"

RUN mkdir -p /root/.cache/pokete && \
    ln -s /root/.cache/pokete /data

COPY . .

VOLUME ["/data"]

CMD [ "python", "./pokete.py" ]
```